src/powershell-unix/powershell-unix.csproj

```diff
@@ -45,6 +45,11 @@
     <DefineConstants>$(DefineConstants);UNIX</DefineConstants>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(Configuration)' == 'LinuxDebug' ">
+    <DebugType>portable</DebugType>
```

Comment:
I believe .NET CLI generates portable pdb by default on Linux/OSX. See https://github.com/OmniSharp/omnisharp-vscode/wiki/Portable-PDBs#how-to-generate-portable-pdbs